assets/sass/components/setup/_googlesitekit-adsense-connect-cta.scss

```diff
@@ -0,0 +1,186 @@
+/**
+ * AdSense Connect CTA styles.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-setup__wrapper--adsense-connect {
+		$bp-nonMobile: $width-tablet + 1 + px;
+
+		.googlesitekit-setup-module--adsense__stage-caption-container {
+			display: grid;
+			grid-template-columns: 1fr;
+			margin: 0;
+
+			@media (min-width: $bp-nonMobile) {
+				// Revert from grid layout to prevent the captions from being overlaid
+				// upon each other.
+				display: block;
+			}
+		}
+
+		.googlesitekit-setup-module--adsense__stage-caption {
+
+			color: $c-subtle;
+			// Set each caption to the same grid cell to overlay them
+			// in order to fade in/out.
+			grid-column-start: 1;
+			grid-row-start: 1;
+			margin: 0;
+			opacity: 0;
+
+			transition-duration: 2.5s;
+			transition-property: border-left, color, opacity;
+			transition-timing-function: ease-in-out;
+
+			h4 {
+				font-size: 18px;
+				font-weight: 400;
+				margin: 0 0 4px;
+
+				transition: inherit;
+			}
+
+			p {
+				font-size: 12px;
+				margin: 0;
+
+				transition: inherit;
+			}
+
+			@media (min-width: $bp-nonMobile) {
```

Comment:
@techanvil Let's move this before the styles for `h4`, [L50](https://github.com/google/site-kit-wp/pull/5336/files#diff-49489c5b6c975530fa4dfea0af9ceae9fae06785debf85cff2a98c299293c717R50)